src/pages/User/components/PullRequests/index.js

```diff
@@ -161,6 +161,22 @@ class PullRequests extends Component {
     return count;
   }
 
+  /**
+   *
```

Comment:
A proper doc block must contain a description. Something like this. 
```suggestion
   * Validates and returns a list of valid pull requests.
```